cmd/fetchd/cmd/gen_asi_upgrade_manifest.go

```diff
@@ -19,9 +19,16 @@ type ASIUpgradeTransfers struct {
 	To       string               `json:"to"`
 }
 
+type ASIUpgradeSupplyMint struct {
+	LandingAddress string      `json:"landing_address"`
+	Amount         types.Coins `json:"amount"`
+	NewSupplyTotal types.Coins `json:"new_supply_total"`
+}
+
 type ASIUpgradeManifest struct {
-	IBC            *ASIUpgradeTransfers `json:"ibc,omitempty"`
-	Reconciliation *ASIUpgradeTransfers `json:"reconciliation,omitempty"`
+	SupplyMint     []ASIUpgradeSupplyMint `json:"supply_mint"`
```

Comment:
1. I'm not quite sure why is this an array. This can be simply directly value type, or pointer to type.
   * In the example below I used the pointer to type in order to have additional degree of freedom - to enable complete exclusion of the Supply data member from json completelly, if really required. Though honestly, it is not required  - it can be directly value type `ASIUpgradeSupplyMint`, since its `Coins` data members are arrays themselves.
2. I would rename this data member to simply `Supply`:

```suggestion
	SupplyMint     *ASIUpgradeSupplyMint `json:"supply_mint"`
```